src/data_info/services/metadata.clj

```diff
@@ -301,13 +305,16 @@
   {:id  uuid
    :uri (build-ore-uri m)})
 
-(defn- build-ore [cm writer agg-path ore-path files avus]
-  (let [ore-file-info? (comp (partial = ore-path) :path)]
+(defn- build-ore [cm writer agg-path ore-path files avus md-path]
+  (let [md-file-info?       (comp (partial = ore-path) :path)
```

Comment:
Should this be `(partial = md-path)`?

Reply:
Ah, yes, you're right. Thanks  for catching that.